src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,159 @@
+package duke.storage;
+
+import duke.TaskList;
+import duke.command.Command;
+import duke.exception.AdditionalException;
+import duke.parser.Parser;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a storage manager that deals with loading tasks from the file and saving tasks in the file.
+ */
```

Comment:
Well done with including javadoc comments across your files to make the code easier to understand1